lib/src/main/kotlin/no/kommune/oslo/origo/maskinporten/MaskinportenKlient.kt

```diff
@@ -33,17 +32,27 @@ class MaskinportenKlient(
 
         log.debug("Henter token fra maskinporten..")
         client.newCall(request).execute().use { response ->
+            try {
             if (!response.isSuccessful) {
                 val resp = response.body?.string()
                 log.warn("Feilet mot maskinporten [{}]: {}", response.code, resp)
-                throw MaskinportenKlientException("Unexpected code $response")
+                val error = jacksonObjectMapper().readValue(resp, MaskinportenError::class.java)
+
+                if(error.error_description.contains("Unknown key identifier", true)){
```

Comment:
Ikke dokumentert i Maskinporten-feilmeldinger, men denne oppstod når jeg først opprettet ny nøkkel for klient, og deretter slettet den gamle lokalt.